spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -99,12 +99,14 @@ class SpyderKernelSpec(KernelSpec, SpyderConfigurationAccessor):
     CONF_SECTION = 'ipython_console'
 
     def __init__(self, is_cython=False, is_pylab=False,
-                 is_sympy=False, **kwargs):
+                 is_sympy=False, is_environment=False,
+                 path_to_environment='', **kwargs):
```

Comment:
Maybe we could change these two `kwargs` to use a single one like `path_to_custom_interpreter=None`?